README.md

```diff
@@ -8,13 +8,13 @@
 
 # Azure Functions .NET Worker
 
-Welcome to the Azure Functions .NET Worker Repository. The .NET Worker provides .NET 5 support in Azure Functions, introducing an **Isolated Model**, running as an out-of-process language worker that is separate from the Azure Functions runtime. This allows you to have full control over your application's dependencies as well as other new features like a middleware pipeline.
+Welcome to the Azure Functions .NET Worker Repository. Since 2020, Azure Functions introduced an **Isolated Worker Model**, running as an isolated language worker that is separate from the Azure Function runtime. This allows you to have full control over your application's dependencies, easily incorporate Dependency Injection as well as other advanced .NET features such as middleware.
```

Comment:
- The second sentence could flow a little more smoothly; maybe something like this?

```suggestion
Welcome to the Azure Functions .NET Worker Repository. Introduced in 2020, Azure Functions' **Isolated Worker Model** moves function execution into a separate language worker process. Isolating function execution from the Azure Functions runtime allows you to have full control over your application's dependencies and easily incorporate advanced .NET features such as middleware and dependency injection.
```